share/locale/tours_fr.ts

```diff
@@ -0,0 +1,344 @@
+    Click &quot;Next&quot; to learn more.</source>
+        <comment>timeline-tour</comment>
+        <translation>Bienvenue dans la Timeline, où vous pouvez avoir un aperçu réduit de votre partition pour pouvoir naviguer plus facilement.
+Cliquez sur &quot;Suivant&quot; pour en savoir plus.</translation>
+    </message>
+    <message>
+        <source>The timeline shows rehearsal marks, changes of tempo, key, and time signature, and other points of interest.
+    You can click anywhere to jump to the corresponding location in your score.</source>
+        <comment>timeline-tour</comment>
+        <translation>La Timeline vous montre les repères, les changements de tempo, les indications de mesure et autres points intéressants.
+Vous pouvez cliquer n&apos;importe où pour aller à l&apos;endroit correspondant dans votre partition.</translation>
+    </message>
+    <message>
+        <source>The section below contains a row for each instrument.
+    Each box within the row represents a measure for that instrument.
+    If the box is highlighted, then there are notes in that measure.
+    Boxes with a thin blue outline represent measures that are currently in view.</source>
+        <comment>timeline-tour</comment>
+        <translation>La section ci-dessous contient une ligne pour chaque instrument.
+Chaque case de la ligne représente une mesure pour cet instrument.
+Si la case est surlignée, c&apos;est qu&apos;il y a des notes dans cette mesure.
+Les cases avec une fine ligne autour représentent les mesures actuellement affichées.</translation>
+    </message>
+    <message>
+        <source>MuseScore supports different types of selections: single, list, and range.
+    Click &quot;Next&quot; to learn more.</source>
+        <comment>select-tour</comment>
+        <translation>MuseScore supporte différents types de sélections : 
+simple, liste et plage.</translation>
+    </message>
+    <message>
+        <source>To make a single selection, click the element you want to select.
+    It becomes highlighted to indicate that it is selected.
+    You can use the left and right cursor keys to move the selection through the notes and rests of your score.
+    Tip: if you press the Alt key (Option on macOS) along with the cursor keys you can move through other elements too,
+    and also move up and down from staff to staff.</source>
+        <comment>select-tour</comment>
+        <translation>Pour effectuer une sélection unique, cliquez sur l&apos;élément que vous souhaitez sélectionner. 
+Il est mis en surbrillance pour indiquer qu&apos;il est sélectionné.
+ Vous pouvez utiliser les touches de curseur gauche et droite pour déplacer la sélection dans les notes et les silences de votre partition.
+ Conseil : si vous appuyez sur la touche Alt (Option sur macOS) en même temps que les touches de curseur, vous pouvez également vous déplacer parmi d&apos;autres éléments, 
+ainsi que de haut en bas d&apos;un personnel à l&apos;autre.</translation>
```

Comment:
Here, the "staff" should be translated to "une portée", not "un personnel"

Reply:
outch ;-)